packages/apps/docs/src/utils/staticGeneration/checkSubTreeForActive.mjs

```diff
@@ -1,7 +1,7 @@
 import { getData } from './getData.mjs';
 import path from 'path';
 
-const omit = (obj, ...keysToOmit) =>
+const omit = (obj, keysToOmit) =>
```

Comment:
this makes sure we need a lot less data in the getstaticprops.
making the website much faster